section3/exercises/arrays.rb

```diff
@@ -25,43 +25,51 @@
 
 # YOU DO: Write code below that will print the number of elements in array of
 # animals from above.
-
+animals.each do |animals|
+  puts "Type of animal : #{animals}"
+end
 
 # YOU DO: Write code that will reassign the last item in the animals
 # array to "Gorilla"
-
+animals[2] = "Gorilla"
 
 # YOU DO: Write code that will add a new animal (type of your choice) to position 3.
-
+animals[3] = "Lion"
 
 # YOU DO: Write code that will print the String "Elephant" in the animals array
-
+print animals[-1]
 
 #-------------------
 # PART 2: Foods: Array Methods
 #-------------------
 
 # YOU DO: Declare a variable that will store an an array of at least 4 foods (strings)
-
+food = ["Banana", "Orange", "Apple", "Peach"]
 
 # YOU DO: Write code below that will print the number of elements in the array of
 # foods from above.
+food.each do |food|
+  puts "Examples of different types of food : #{food}"
+end
 
 
 # YOU DO: Write code below that uses a method to add "broccoli" to the foods array and
 # print the changed array to verify "broccoli" has been added
-
+food[4] = "Broccoli"
```

Comment:
Nice. How else can you add an element to an array (especially if you don't know/care what the 'new' last index is?)